### DISCUSSION TITLE
Hyper-3.1.3. - Next tab shortcut missing - Fixed

### DISCUSSION THREAD

**Discussion by vennai** [vercel/hyper#5964] [Q&A] [answered]
If you've upgraded and lost access to the next tab shortcut:
Right click -> Preferences and add the following line under keymaps: 

```
 keymaps: {
    'tab:next': 'ctrl+tab'
  },
```

**Comment by churichard** [comment, marked as answer]
Thank you! This was exactly what I was looking for 🙏

I added:
```js
keymaps: {
  'tab:next': 'ctrl+tab',
  'tab:prev': 'ctrl+shift+tab'
},
```

**Reply by enti-re** [reply]
Thank you✨✨
